packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -56,15 +56,13 @@ export default class WalletService {
   }
 
   public create = ({ name, keystore }: { name: string; keystore: Keystore }): Wallet => {
-    const id = walletStore.saveWallet({ id: v4(), name, keystore })
-    if (id) {
-      const storedWallet = walletStore.getWallet(id)
-      return {
-        id,
-        name: storedWallet.name,
-      }
+    const id = v4()
+    walletStore.saveWallet({ id, name, keystore })
+    const storedWallet = walletStore.getWallet(id)
```

Comment:
The `storedWallet` is equal to `{id, name, keystore}`. Refactor it later.

Reply:
ok